server/server-stats.cpp

```diff
@@ -418,6 +418,36 @@ struct WorkerStatsBundle : EnumTable<E, std::array<std::atomic<typename E::StatT
   }
 };
 
+template<typename T>
+struct WorkerSamples : vk::not_copyable {
+public:
+  void recalc(size_t worker_index) {
+    percentiles.update_percentiles(samples.begin(), samples.begin() + worker_index);
+  }
+
+  Percentiles<T> percentiles;
+  std::array<T, WorkersControl::max_workers_count> samples{};
```

Comment:
What exactly will `samples` store and why is it needed? One last stat value for each worker process?